maintainers/scripts/check-maintainer-github-handles.sh

```diff
@@ -62,5 +61,3 @@ export -f checkUser
 nix-instantiate -A lib.maintainers --eval --strict --json \
     | jq -r '.[]|.github|select(.)' \
     | parallel -j5 checkUser
-
-# parallel -j100 checkUser ::: "eelco" "profpatsch" "Profpatsch" "a"
```

Comment:
That should probably be kept.

Reply:
If you can explain why, I could include that in the comment. As it stands, I feel like that just looks like accidentally committed test code.

Reply:
I think that is when you want to manually check only a few users.

Reply:
Documented, cheers.